### PR TITLE
Metawolf:  Add recipe autocomplete and some small fixes here and there

### PR DESCRIPTION
**Pull request by giovannt0**
- Add recipe autocomplete: the entire `show` command now supports autocomplete. `set` too.
- Trim the timestamp output to a less long precision
- Hide cmd2 commands from the help menu, and be a little more verbose about what metawolf's command do. Also, always display the verbose help.
- Move outputID column next to commandID under `show running`
- Add some additional guidance output for users
- Add a status attribute to the process class

### CODE REVIEW

File: dftimewolf/metawolf/output.py [hunk 1]
```diff
@@ -216,22 +223,30 @@ def Status(self) -> str:
...
-      return MetawolfOutput.Color('Interrupted', RED)
+      self.status = MetawolfOutput.Color('Interrupted', RED)
+      return self.status
 
     if return_code == -1:
-      return MetawolfOutput.Color('Unknown', BLUE)
+      if not self.status:
+        # No previous known state from file.
+        self.status = MetawolfOutput.Color('Unknown', BLUE)
+        return self.status
```
Comment (tomchop) [inline dftimewolf/metawolf/output.py:238]: superfluous return, given that you have the one right below?

File: tests/cli/main_tool.py [hunk 2]
```diff
@@ -16,7 +16,11 @@ class MainToolTest(unittest.TestCase):
   def setUp(self):
     self.tool = dftimewolf_recipes.DFTimewolfTool()
     self.tool.LoadConfiguration()
-    self.tool.ReadRecipes()
+    try:
+      self.tool.ReadRecipes()
+    except KeyError:
```
Comment (tomchop) [inline tests/cli/main_tool.py:21]: Are you still running into this problem despite deregistering the recipes in TearDown?

File: dftimewolf/metawolf/metawolf.py [hunk 3]
```diff
@@ -338,28 +375,41 @@ def do_run(self, _: cmd2.Statement) -> None:
...
+  def do_show(self, args: argparse.Namespace) -> None:
+    """Show sessions, recipes, runs and outputs. `help show` for details.
+
+    Possible choices: [`-recipes`, `-recipe recipe_name`, `-sessions`,
+    `-running`, `-output output_id`]. `recipes` shows the user the available
+    DFTimewolf recipes, while `recipe recipe_name` shows the details of a
+    given recipe. `sessions` shows the currently available Metawolf's sessions
+    and any recipe in use, along with information about the recipe's state.
+    `running` displays running jobs and their state. `output output_id` prints
+    to STDOUT the output of the matching output_id. Autocomplete is available.
```
Comment (tomchop) [inline dftimewolf/metawolf/metawolf.py:400]:
```suggestion
    Possible choices: [`-recipes`, `-recipe recipe_name`, `-sessions`,
    `-running`, `-output output_id`]. `recipes` shows the user the available
    DFTimewolf recipes, while `recipe recipe_name` shows the details of a
    given recipe. `sessions` shows the currently available Metawolf sessions
    and any recipe in use, along with information about the recipe's state.
    `running` displays running jobs and their state. `output output_id` writes
    the output of the matching output_id to STDOUT. Autocomplete is available.
```

**Review by tomchop — CHANGES_REQUESTED** [review]
Minor nits

File: tests/cli/main_tool.py (hunk 2 above)
Comment (giovannt0) [reply tests/cli/main_tool.py:21]: Yes :(

**Review by tomchop — APPROVED** [review]

**Pull request merged**